pkg/app/run.go

```diff
@@ -26,6 +26,10 @@ func NewRun(st *state.HelmState, helm helmexec.Interface, ctx Context) *Run {
 		panic("Assertion failed: helmexec.Interface must not be nil")
 	}
 
+	if !helm.IsHelm3() {
+		panic("helmfile has deprecated helm2 since v1.0")
```

Comment:
since v0.150.0. not panic. we can use `os.Exit`? WDYT? @mumoshu @xiaomudk

Reply:
Good idea, panic message is not very friendly